src/Akeneo/Connectivity/Connection/back/Application/Webhook/Service/EventSubscriptionLogInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Application\Webhook\Service;
+
+use Akeneo\Connectivity\Connection\Application\Webhook\Log\EventSubscriptionSendApiEventRequestLog;
+use Akeneo\Platform\Component\EventQueue\EventInterface;
+
+/**
+ * @author Pierre Jolly <pierre.jolly@akeneo.com>
+ * @copyright 2021 Akeneo SAS (http://www.akeneo.com)
+ * @license http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+interface EventSubscriptionLogInterface
```

Comment:
```suggestion
interface EventsApiLogger
```
Naming suggestion, let's see what others think

Reply:
Do you see in this interface that we are breaking the SRP? If you type-hint the service you inject in all the classes with this interface, it means you enforce to inject in a class that only needs to log a reach limit error, another class that MUST be able to log a reach limit error AND to log an event build error AND to log an event data builder error AND to ... etc. 
What I want to explain is: I'm ok to gather all the methods to log in to one class (even if I'm not convinced of the real benefit), but I'm not ok to create only one interface to gather them. You should create as many interfaces as you have business cases in order to respect the SRP.

Reply:
Yes you're right I'll fix this